packages/side-nav/src/SideNav.story.tsx

```diff
@@ -56,15 +60,15 @@ const realmAppContainer = css`
   flex-grow: 1;
 `;
 
-const mongoNavStyles = css`
+const mongoNavStyles = (darkMode: boolean) => css`
```

Comment:
should we have baseMongoNavStyles and then use an object to set the variable properties?

Reply:
cc @spark33 @shaneeza

Reply:
seeing as it was just for the story I didn't bother to create the object but I can update this